pkg/authorization/api.md

```diff
@@ -0,0 +1,83 @@
+# Docker Authorization Plug-in API
```

Comment:
I think this should be moved to a new `/docs/extend/reference` directory.  Leaving it in the package makes it hard to find and not so great for SEO